### PR TITLE
sdk: get rid of float conversions in GetCoreOSAge

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

File: sdk/version.go [hunk 1]
```diff
@@ -68,5 +68,5 @@ const (
 
 // GetCoreOSAge returns the number of days since the CoreOS epoch.
 func GetCoreOSAge() int64 {
-	return int64(time.Since(time.Unix(CoreOSEpoch, 0)).Seconds()) / 86400
+	return int64(time.Since(time.Unix(CoreOSEpoch, 0)) / (86400 * time.Second))
```
Comment (marineam) [inline sdk/version.go]: Should be able to drop the int64 cast right?

Comment (mischief) [inline sdk/version.go]: only if we want to return time.Duration.

Comment (marineam) [inline sdk/version.go]: Ah, right. In that case the intent would be clearer if the whole expression was wrapped in the cast, not just time.Since.

Comment (marineam) [inline sdk/version.go]: Er, wait. nm

I got my parrens mismatched, you are doing that. this is why I don't write LISP

**Comment by marineam** [comment]
LGTM

**Pull request merged**